src/main/java/com/github/ermay12/regex/Regex.java

```diff
@@ -241,6 +244,7 @@
   private Pattern pattern;
   private String rawRegex;
   private final String privatesyncobj = "";
+  protected Map<CapturingGroup, Integer> groupToIndex = new HashMap<>();
 
   /*
```

Comment:
Please modify the class-level documentation to reflect the absence of named capturing groups!